### DISCUSSION TITLE
Group access

### DISCUSSION THREAD

**Discussion by NoipName** [thorsten/phpMyFAQ#2656] [Ideas]
Hello all !

I installed recently PhpMyFAQ on my server. I connected Azure for MyFAQ and it works nicely !
The gestion access userto make some action  is works great but is a little bit ennoying if we need to give same access for 200 peoples :D 

My idea is for the new patch update, can we add the notion "create group access", like this but for group ?
Create group, add user in the group and give permission for group and that applicate for all user
![image](https://github.com/thorsten/phpMyFAQ/assets/145665195/e90b464f-8282-405f-9fe4-84095ce28611)

Thanks you for your answer 👍

**Comment by thorsten** [comment]
It's already possible. You can create a group with the correct permissions and set the flag "auto join". When all new users should automatically join this group.

**Reply by NoipName** [reply]
Oh ok thank you, but that option itsn't by default in administration interface with the version 3.2.1 ?
![image](https://github.com/thorsten/phpMyFAQ/assets/145665195/1dd0a059-0995-4dd3-88a3-b46b8c3ea3be)

In the left only "Users" apparear, maybe i forget something but i didn't find "create group/add group" :(

**Reply by thorsten** [reply]
ah, you have to enable group support under Admin -> Configuration -> Security with the permission level "medium"
<img width="503" alt="Bildschirmfoto 2023-09-22 um 15 51 53" src="https://github.com/thorsten/phpMyFAQ/assets/45284/5b82d4a0-f7df-4fbf-b2a7-648afad4a78e">

**Reply by NoipName** [reply]
Ouhhh niceee :D thanks you
